physlr/physlr.py

```diff
@@ -1462,19 +1478,23 @@ def physlr_split_minimizers(self):
             sys.exit(msg)
         g = self.read_graph([self.args.FILES[0]])
         bxtomxs = self.read_minimizers([self.args.FILES[1]])
+        bxtocount = self.count_molecules_per_bx(bxtomxs)
```

Comment:
Hmm I'm not sure this works exactly as we want it to... For `count_molecules_per_bx`, it expects a dictionary of molecule -> minimizers, but you are giving it bx -> minimizers. So I think it would end up giving you an empty Counter?

Reply:
fixed